rustfmt-config/src/config_type.rs

```diff
@@ -365,12 +373,50 @@ macro_rules! create_config {
                     self.set().width_heuristics(WidthHeuristics::null());
                 }
             }
+
+            fn set_license_template(&mut self) {
+                let license_template_path = self.license_template_path();
```

Comment:
Are you accounting for a possibly empty license_template_path somewhere?

Reply:
No, well-spotted! I didn't realize calling rustfmt with no config file is not quite the same as calling with a config file which doesn't specify the `license_template_path`.

Reply:
done